src/event.js

```diff
@@ -591,17 +593,17 @@ function leverageNative( el, type, expectSync ) {
 
 			// If this is a native event triggered above, everything is now in order
 			// Fire an inner synthetic event with the original arguments
-			} else if ( saved ) {
+			} else if ( saved.length ) {
 
 				// ...and capture the result
-				dataPriv.set( this, type, jQuery.event.trigger(
+				dataPriv.set( this, type, { value: jQuery.event.trigger(
```

Comment:
This would IMO look more readable with `value` moved to the next line. I don't recognize this way of writing objects from our code base.

Reply:
Agreed.